dynamic/provider_test.go

```diff
@@ -408,11 +408,13 @@ func TestSchemaGeneration(t *testing.T) { //nolint:paralleltest
 	skipWindows(t)
 
 	t.Run("unparameterized", func(t *testing.T) {
+		t.Skipf("Skipping due to https://github.com/pulumi/pulumi/issues/19274")
```

Comment:
Does it mean all dynamically bridged providers are disabled by this change?

Reply:
No, thankfully! This tests specifically checks for the _unparameterized_ provider, whose package name is `terraform-provider`. The other tests, which use parameterized providers, all pass.

The issue here is that the package name is unusual in that it has a hyphen, which chokes during validation.